bot/bot.ttl

```diff
@@ -34,42 +34,73 @@
 bot:Building a owl:Class ;
         owl:sameAs      dbo:Building ;
         rdfs:label      "Building"@en ,
-						"Gebäude"@de ,
-                        "Bygning"@da ;
+			"Gebäude"@de ,
+                        "Bygning"@da ,
+			"Edificio"@it ;
         rdfs:comment    "An independent unit of the built environment with a characteristic spatial structure, intended to serve at least one function or user activity [ISO 12006-2:2013]"@en ,
-						"Bauwerk hauptsächlich zum Zweck des Schutzes für seine Bewohner und die darin aufbewahrten Gegenstände; im Allgemeinen teilweise oder ganz geschlossen und ortsfest [ISO 6707-1:2014]"@de ,
-                        "En uafhængig del af det byggede miljø med en karakteristisk rumlig struktur, der understøtter mindst én funktion eller brugeraktivitet"@da .
+			"Bauwerk hauptsächlich zum Zweck des Schutzes für seine Bewohner und die darin aufbewahrten Gegenstände; im Allgemeinen teilweise oder ganz geschlossen und ortsfest [ISO 6707-1:2014]"@de ,
+                        "En uafhængig del af det byggede miljø med en karakteristisk rumlig struktur, der understøtter mindst én funktion eller brugeraktivitet"@da ;
+	rdfs:subClassOf 
+		[rdf:type owl:Restriction ;
+			owl:onProperty bot:hasElement ;
+			owl:allValuesFrom bot:Element
+		] ,
```

Comment:
the range of bot:hasElement is already bot:Element. This axiom adds no semantics